lib/model/QubitNote.php

```diff
@@ -22,6 +22,24 @@ class QubitNote extends BaseNote
     // Flag for updating search index on save or delete
     public $indexOnSave = true;
 
+    public function __get($name)
+    {
+        $args = func_get_args();
+
+        $options = [];
+        if (1 < count($args)) {
+            $options = $args[1];
+        }
+
+        if ('type' === $name && true === $options['sourceCulture']) {
+            $sql = 'SELECT term_i18n.name from note join term_i18n on  term_i18n.id = note.type_id where term_i18n.id=? and note.id=? and term_i18n.culture=?';
```

Comment:
Nitpick: breaking the SQL into multiple lines and capitalizing SQL keywords ("from", "where", etc.) would make it more readable.

```
$sql = 'SELECT term_i18n.name from note
    join term_i18n on term_i18n.id = note.type_id
    where term_i18n.id=?
    and note.id=? and term_i18n.culture=?';
```

Reply:
Fixed these!